gnomad_qc/v4/create_release/create_custom_liftover.py

```diff
@@ -0,0 +1,131 @@
+" Script to create custom liftover file for genes CBS, KCNE1, and CRYAA"
+import argparse
+import logging
+
+import hail as hl
+
+from gnomad.resources.resource_utils import (
+    MatrixTableResource,
+    PedigreeResource,
+    TableResource,
+    VersionedMatrixTableResource,
+    VersionedPedigreeResource,
+    VersionedTableResource,
+)
+from gnomad_qc.v2.resources.basics import get_gnomad_liftover_data_path
+from gnomad_qc.v4.resources.constants import CURRENT_RELEASE, RELEASES
+from gnomad_qc.v4.resources.release import (
+    # release_lof,
+    _release_root,
+    release_sites,
+    # release_summary_stats
+)
+
+
+logging.basicConfig(
+    format="%(asctime)s (%(name)s %(lineno)s): %(message)s",
+    datefmt="%m/%d/%Y %I:%M:%S %p",
+)
+logger = logging.getLogger("subset")
+logger.setLevel(logging.INFO)
+
+
+def get_custom_liftover_three_genes_path(
+    release_version: str = CURRENT_RELEASE,
+    test: bool = False,
+    data_type: str = "genomes",
+) -> str:
+    """
+    Retrieve path for the liftover table containing three genes of interest.
+
+    :return: Combined custom liftover table path.
+    """
+    return (
+        f"{_release_root(version=release_version, test=test, data_type=data_type)}/gnomad.{data_type}.v{release_version}_custom_liftover_three_genes.ht"
+    )
+
+
+def main(args):
+    # Read in liftover file
+    liftover_ht = hl.read_table(
+        get_gnomad_liftover_data_path(data_type="exomes", version="2.1.1").path
+    )
+
+    # Create and filter to genes of interest
+    # Current coordinates grabbed from gnomAD v4 browser, have reached out to browser team about something more professional.
```

Comment:
thank you for both of these! Vep's transcript consequence field (with gene_symbol in it) is a little crowded, so I didn't see this and didn't know it was an option! I might use that going ahead, since it seems like the easiest approach. The browser coordinates do include UTRs, which I believe the audience for these files would want, so let me check and make sure those are included in here.

Reply:
filtered using gene_symbol in VEP in new commit. Definitely should be the approach, but I won't resolve this as a note that we should triple-check these are what we want at some point before uhh releasing this